components/edit-collective/Menu.js

```diff
@@ -155,7 +155,11 @@ const sectionsDisplayConditions = {
   [EDIT_COLLECTIVE_SECTIONS.UPDATES]: isFeatureAllowed(FEATURES.UPDATES),
   [EDIT_COLLECTIVE_SECTIONS.CONVERSATIONS]: isFeatureAllowed(FEATURES.CONVERSATIONS),
   [EDIT_COLLECTIVE_SECTIONS.EXPENSES]: isType(CollectiveType.COLLECTIVE),
-  [EDIT_COLLECTIVE_SECTIONS.EXPORT]: isType(CollectiveType.COLLECTIVE),
+  [EDIT_COLLECTIVE_SECTIONS.EXPORT]: isOneOfTypes(
+    CollectiveType.COLLECTIVE,
+    CollectiveType.ORGANIZATION,
+    CollectiveType.USER,
```

Comment:
What if Organization or User are not Hosts?

Reply:
I will look into this, and i'm guessing there is a new type `CollectiveType.HOST`.